.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
```

Comment:
This is what I mean: https://github.com/fastify/fastify/blob/7db09f107d83e6f85907e43199ac945e4d761e52/.github/workflows/ci.yml#L3